src/price_data.py

```diff
@@ -646,7 +659,10 @@ def get_price(
             # Do not save price in database.
             price = self.__mean_price_db(db_path, tablename, utc_time)
 
-        return price
+        if inverted:
+            return decimal.Decimal(1 / price)
```

Comment:
You can use `misc.reciprocal` for that.
https://github.com/provinzio/CoinTaxman/blob/main/src/misc.py#L90-L91